src/GraphQL.Tests/Bugs/Bug1772.cs

```diff
@@ -45,9 +45,10 @@ public async Task DocumentExecuter_throws_for_invalid_operation(string operation
             result.Data.ShouldBeNull();
             result.Errors.ShouldNotBeNull();
             result.Errors.Count.ShouldBe(1);
-            result.Errors[0].Message.ShouldBe($"Query does not contain operation '{operationName}'.");
+            result.Errors[0].Message.ShouldBe("Error executing document.");
             result.Errors[0].InnerException.ShouldNotBeNull();
             result.Errors[0].InnerException.ShouldBeOfType<InvalidOperationException>();
+            result.Errors[0].InnerException.Message.ShouldBe($"Query does not contain operation '{operationName}'.");
```

Comment:
Note that this particular test produces an `InvalidOperationException` because an invalid operation name was passed to `ExecuteAsync` within `ExecutionOptions`.  It is not invalid data within the query or variables; it is an incorrect parameter passed to the execution engine.